### PR TITLE
Wewewe

### PR DESCRIPTION
**Pull request by Therapist2**

### CODE REVIEW

File: hero.c [hunk 1]
```diff
@@ -163,6 +165,19 @@ void hero_init_lili(struct hero *hero){
...
 }
 
+void hero_init_allahrak(struct hero *hero){
+    int base_attack = hero->basic_attack_damage;
+
+    hero->model = 'l';
+
+    hero->hp_max = (hero->hp_max * 8) / 10;
+
+    hero->basic_attack_damage = (base_attack * 4) / 10;
```
Comment (kuche1) [inline hero.c:175]: extreme damaga???

File: hero.c [hunk 2]
```diff
@@ -163,6 +165,19 @@ void hero_init_lili(struct hero *hero){
...
+
+    hero->heal_ability_range = 2;
```
Comment (kuche1) [inline hero.c:177]: alahrak heals

File: hero.c [hunk 3]
```diff
@@ -163,6 +165,19 @@ void hero_init_lili(struct hero *hero){
     hero->heal_ability_amount = (base_attack * 6) / 10;
 }
 
+void hero_init_allahrak(struct hero *hero){
+    int base_attack = hero->basic_attack_damage;
+
+    hero->model = 'l';
```
Comment (kuche1) [inline hero.c:171]: woman

File: hero.c (hunk 1 above)
Comment (Therapist2) [reply hero.c:175]: extreme damagato

File: hero.c (hunk 2 above)
Comment (Therapist2) [reply hero.c:177]: ٱللَّٰه

**Comment by kuche1** [comment]
unacceptable
allahrak will never forgive

**Pull request closed without merging**